resources/lib/library.py

```diff
@@ -1494,8 +1494,8 @@ def explorer( self, history, location, label, thumbnail, itemType, isWidget = Fa
                     listitem.setProperty( "widgetPath", location )
 
                 elif itemType == "32011" or itemType == "32019" or itemType == "32073":
-                    action = 'ActivateWindow(10501,"' + location + '",return)'
-                    listitem.setProperty( "windowID", "10501" )
+                    action = 'ActivateWindow(10502,"' + location + '",return)'
+                    listitem.setProperty( "windowID", "10502" )
```

Comment:
no, it doesnt make a practical difference, but it's more readable and less likely to get changed in the future.
Just a recommendation, if you want to stick with ids then feel free to do so.